styles/index.css

```diff
@@ -15,21 +15,118 @@ body {
 
 h1 {
   text-align: center;
-  border: 1px solid black;
+  border: 1px solid #ee6c4d;
   margin: 0;
   padding: 1.25rem;
   text-transform: uppercase;
 }
 
+/* UTILITY */
+
+.mx-1rem {
+  margin: 0 1rem;
+}
+
+/* COMPONENTS*/
+
+/* QUESTION CARDS*/
+
 .card {
-  border: 1px solid black;
+  border: 1px solid #ee6c4d;
+  padding: 0.5rem;
 }
 
-.navigation {
-  border: 1px solid black;
+.card + .card {
+  margin-top: 1rem;
+}
+
+.questions {
+  width: 90%;
```

Comment:
Be careful with percentage. The 10% on the right could become too big or too small on other devices.